frontend/src/components/search/SaveSearchModal.tsx

```diff
@@ -85,7 +83,10 @@ function SuccessContent({
   );
 }
 
-export function SaveSearchModal() {
+// note that this is client component, but not marked as "use client" because to do so
+// would cause an error when passing the function prop from parent client component see:
+// https://github.com/vercel/next.js/discussions/46795
```

Comment:
Interesting, makes sense, thanks for the note.

![Themoreyouknow](https://github.com/user-attachments/assets/76611019-620e-420f-9a38-2327d7501117)